cmd/terraform-registry/main.go

```diff
@@ -81,12 +84,15 @@ func init() {
 	flag.StringVar(&tlsCertFile, "tls-cert-file", "", "")
 	flag.StringVar(&tlsKeyFile, "tls-key-file", "", "")
 	flag.StringVar(&storeType, "store", "", "Store backend to use (choices: github, s3)")
+	flag.BoolVar(&providerStoreEnabled, "provider-store-enabled", false, "Enable discovery of providers")
```

Comment:
I wonder if this instead can be translated into a string value `-provider-store`, just like `-store` is currently set up today. Then let's put on the roadmap to rename `-store` to `-module-store`.

Reply:
I like it!